src/components/Common/AppContainer.tsx

```diff
@@ -35,7 +35,9 @@ export const AppContainer = ({ children }: AppContainerProps) => {
       pathname === '/mypage' ||
       pathname === '/review' ||
       pathname === '/paymentDetail' ||
-      pathname.includes('/chat/')
+      pathname.includes('/chat/') ||
+      pathname === '/minder/consult' ||
+      pathname.includes('/minder/open-consult')
```

Comment:
이 부분 편지, 채팅 부분이 경계가 사라져서, 공개 상담에만 적용되어야 할 것 같습니다!
```suggestion
      pathname.includes('/chat/') ||
      pathname.includes('/open-consult')
```
이런식으로???

https://www.figma.com/design/cZgzroEqmJBGoHvRsTLzmX?node-id=4856-66494#824699681

<img width="369" alt="image" src="https://github.com/sharemindteam/sharemind-client/assets/64120854/7ce67bbc-86b8-463f-9315-e85f21ed84f8">

Reply:
넵 피그마 코멘트 확인했습니다! 
 제안해주신 코드에서는 상담탭(/consult) 의 공개상담이 Grey가 되지않아서, useLoaction() 훅에서 search 키 받아온 후 조건문 한줄 더 추가해서 구현했습니다